doc/client_lua_api.txt

```diff
@@ -709,7 +709,12 @@ Call these functions only at load time!
     * `spec` is a `SimpleSoundSpec`
     * `parameters` is a sound parameter table
 * `minetest.sound_stop(handle)`
-
+### Key Input
+* `minetest.register_on_keydown(func(key))`
+    * Called when player presses a key.
+    * If the player holds a key down, it'll be called multiple times.
+* `minetest.register_on_keyup(func(key))`
+    * Called when player releases a key (stops holding it).
```

Comment:
Needs reference to the key values http://irrlicht.sourceforge.net/docu/namespaceirr.html#a54da2a0e231901735e3da1b0edf72eb3